docs/rules/array-bracket-spacing.md

```diff
@@ -47,7 +47,7 @@ var arr = [ 'foo', 'bar' ];
 var arr = ['foo', 'bar' ];
 var arr = [ ['foo'], 'bar'];
 var arr = [[ 'foo' ], 'bar'];
-var arr = ['foo',
+var arr = [ 'foo',
```

Comment:
This is invalid for `never`, since there is a space after `[`, unlike line 73.
This same code is valid for `always` (line 119). Is that the one you are referring to?